examples/fishjam-chat/screens/RoomScreen/ToggleAppScreenShare.ios.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback } from 'react';
+import { roomScreenLabels } from '../../types/ComponentLabels';
+import { InCallButton } from '../../components';
+import { useAppScreenShare } from '@fishjam-cloud/react-native-client';
+
+export const ToggleAppScreenButton = () => {
+  const iosAppScreenShare = useAppScreenShare();
```

Comment:
**suggestion:** If it's not optional any more, I guess you can use `{ toggleAppScreenShare }`